src/fibonacci.js

```diff
@@ -0,0 +1,53 @@
+export default function fibonacci(base,number){
+  if(base === "fib"){
+    return decify(number)
+  }
+  else if(base === "decimal"){
```

Comment:
no need to use ```else if``` since you have a return in your first if statement

just use ```if``` again.